app/views/books/index.html.erb

```diff
@@ -1,12 +1,19 @@
 
+<h1 class="text-center">Book Library</h1>
+
 <div class="container container-height">
-  <h1>Book Library</h1>
-  <% @books.each do |book|%>
-  <div class="col-xs-4">
-    <h2><%= book.title %></h2>
-    <%= image_tag book.cover.url(:thumb) %>
-    <h3>buy it today!</h3>
-    <h3><%= book.url %></h3>
+  <div class="col-xs-offset-2">
+    <% @books.each do |book|%>
+      <a href="/books/<%= book.id %>">
+        <div class="col-xs-4 book-preview text-center">
+          <h4><%= book.title %></h4>
+          <% if book.cover.url(:thumb) == "/images/thumb/missing.png" %>
+            <%= image_tag "missing_cover.jpg" %>
+          <% else %>
+            <%= image_tag book.cover.url(:medium) %>
+          <% end %>
+        </div>
+      </a>
+    <% end %>
   </div>
-  <% end %>
 </div>
```

Comment:
I realize that the chances of the database NOT having a book for the library is slim... HOWEVER it is possible. please put in nil guards (I feel like we have already done this? maybe in the next PR that I haven't looked at)
![screen shot 2017-06-15 at 7 53 30 pm](https://user-images.githubusercontent.com/7662875/27206286-876ec99c-5204-11e7-8611-b83451834e86.png)